samples/bot-task-module/csharp/README.md

```diff
@@ -61,10 +62,10 @@ the Teams service needs to call into the bot.
    - Navigate to `samples/bot-task-module/csharp` folder
    - Select `TeamsTaskModule.csproj` or `TeamsTaskModule.sln`file
 
-1) Update the `appsettings.json` configuration for the bot to use the MicrosoftAppId, MicrosoftAppPassword, MicrosoftAppTenantId generated in Step 2 (App Registration creation). (Note the App Password is referred to as the "client secret" in the azure portal and you can always create a new client secret anytime.)
-    - Set "MicrosoftAppType" in the `appsettings.json`. (**Allowed values are: MultiTenant(default), SingleTenant, UserAssignedMSI**)
+1) Update the `appsettings.json` configuration for the bot to use the `<<MicrosoftAppId>>`, `<<MicrosoftAppPassword>>`, `<<MicrosoftAppTenantId>>` generated in Step 2 (App Registration creation). (Note the App Password is referred to as the "client secret" in the azure portal and you can always create a new client secret anytime.)
+    - Set `<<MicrosoftAppType>>` in the `appsettings.json`. (**Allowed values are: MultiTenant(default), SingleTenant, UserAssignedMSI**)
 
-    - Set "BaseUrl" in the `appsettings.json` as per your application like the ngrok forwarding url (ie `https://xxxx.ngrok.io`) after starting ngrok
+    - Set `<<BaseUrl>>` in the `appsettings.json` as per your application like the ngrok forwarding url (ie `https://xxxx.ngrok.io`) after starting ngrok
```

Comment:
You did not mention valid-domain update steps in readme for manifest but place holder is also differnt:
![image](https://user-images.githubusercontent.com/59052547/209147783-0b3d029d-cdb8-4dfb-bc82-643ee81a0024.png)

Reply:
Updated Domain Name in Manifest Section